.github/ISSUE_TEMPLATE/2-PROXY.md

```diff
@@ -0,0 +1,36 @@
+---
```

Comment:
proxy is a bit of a weird name

Reply:
yeah I had originally wanted to write an issue for some specific failures but realized I didn't know exactly where to point for docs like I wanted...